resources/js/components/Elements/Slider.vue

```diff
@@ -102,6 +118,19 @@ export default {
                 ? this.slider.scrollTo(0, this.slider.children[index]?.offsetTop)
                 : this.slider.scrollTo(this.slider.children[0]?.offsetWidth * index, 0)
         },
+
+        handleLoop() {
+            if (this.currentSlide + 1 === this.slidesTotal - 1) {
+                Array.from(this.chunk.children).forEach((child) => {
+                    this.slider.appendChild(child.cloneNode(true))
+                })
+            }
```

Comment:
given that slidesTotal is static, this seems like it'll continuously create duplicates if you go back and forth over the first loop point? I feel like this should be `=== (this.slider.children?.length ?? 1) - this.slidesVisible`

It's tricky to imagine by just reading the code so I could be wrong here :)

Reply:
slidesTotal is no longer static. In the method initSlider this will be set: https://github.com/rapidez/core/pull/309/files#diff-2d53f035dc6e0201915f81219d324c2479c3acd61fde0b67ffbc091205965494R76. Then I added a watcher for currentSlide where initSlider will be triggered: https://github.com/rapidez/core/pull/309/files#diff-2d53f035dc6e0201915f81219d324c2479c3acd61fde0b67ffbc091205965494R144. So if currentSlide changes, the slidesTotal will be set again.

Reply:
Ah, the second part is what I missed 😇